### PR TITLE
Refine statements pages head

### PR DESCRIPTION
**Pull request by baurine**
What did:

- Fix FabricRoot run twice and styles bundled twice
- Extract EllipsisText
- Refine statement overview and detail page head

Screenshots:

WX20200415-151440@2x

WX20200415-151503@2x

### CODE REVIEW

File: ui/packages/statement/src/translations/zh-CN.yaml [hunk 1]
```diff
@@ -1,5 +1,8 @@
 statement:
   nav_title: SQL 语句分析
+  pages:
+    overview: SQL 语句总览
```
Comment (breeswish) [inline ui/packages/statement/src/translations/zh-CN.yaml:4]: For entry pages, there should be no titles

File: ui/packages/dashboard_components/src/EllipsisText/index.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+}
+
+export default function EllipsisText({
+  text,
+  className,
+  ...rest
+}: IEllipsisTextProps) {
+  return (
+    <div className={cx(styles.ellipsis_text, className)} {...rest}>
+      {text}
```
Comment (breeswish) [inline ui/packages/dashboard_components/src/EllipsisText/index.tsx:17]: I discovered that we can actually use https://ant.design/components/typography/#Typography.Text

Comment (baurine) [reply ui/packages/dashboard_components/src/EllipsisText/index.tsx:17]: awesome, let me have a try!

Comment (baurine) [reply ui/packages/dashboard_components/src/EllipsisText/index.tsx:17]: unfortunately, didn't get the expected result:

image

File: ui/packages/statement/src/translations/zh-CN.yaml (hunk 1 above)
Comment (baurine) [reply ui/packages/statement/src/translations/zh-CN.yaml:4]: Removed:

image

**Comment by baurine** [comment]
Updated screenshots:

image

image

File: ui/packages/dashboard_components/src/EllipsisText/index.tsx (hunk 2 above)
Comment (breeswish) [reply ui/packages/dashboard_components/src/EllipsisText/index.tsx:17]: Sad news!

File: ui/packages/dashboard_components/src/EllipsisText/index.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+import cx from 'classnames'
+import styles from './index.module.less'
+
+export interface IEllipsisTextProps
+  extends React.HTMLAttributes<HTMLDivElement> {
+  text: string
+}
+
+export default function EllipsisText({
+  text,
```
Comment (breeswish) [inline ui/packages/dashboard_components/src/EllipsisText/index.tsx:11]: prefer to use children

Comment (baurine) [reply ui/packages/dashboard_components/src/EllipsisText/index.tsx]: done!

**Pull request merged**